.github/workflows/go.yml

```diff
@@ -1,318 +1,326 @@
-    timeout-minutes: 60
-    strategy:
-      fail-fast: false
-      matrix:
-        go: [1.17, 1.18]
-        include:
-          - go: 1.17
-            staticcheck: v0.2.2
-          - go: 1.18
-            staticcheck: latest
-    env:
-      ARROW_GO_TESTCGO: "1"
-    steps:      
-      - name: Checkout Arrow
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-          submodules: recursive
-      - name: Install go
-        uses: actions/setup-go@v3
-        with:
-          go-version: ${{ matrix.go }}
-          cache: true
-          cache-dependency-path: go/go.sum
-      - name: Brew Install Arrow
-        shell: bash
-        run: brew install apache-arrow
-      - name: Install staticcheck
-        run: go install honnef.co/go/tools/cmd/staticcheck@${{ matrix.staticcheck }}
-      - name: Build
-        shell: bash
-        run: ci/scripts/go_build.sh $(pwd)
-      - name: Test
-        shell: bash
-        run: ci/scripts/go_test.sh $(pwd)
-
-  windows-mingw:
-    name: AMD64 Windows MinGW ${{ matrix.mingw-n-bits }} CGO
-    runs-on: windows-2019
-    if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
-    timeout-minutes: 60
-    strategy:
-      fail-fast: false
-      matrix:
-        mingw-n-bits:
-          #- 32 runtime handling for CGO needs 64-bit currently
-          - 64
-    env:
-      ARROW_GO_TESTCGO: "1"
-      MINGW_LINT: "1"
-    steps:
-      - name: Disable Crash Dialogs
-        run: |
-          reg add `
-            "HKCU\SOFTWARE\Microsoft\Windows\Windows Error Reporting" `
-            /v DontShowUI `
-            /t REG_DWORD `
-            /d 1 `
-            /f
-      - name: Checkout Arrow
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-          submodules: recursive
-      - uses: msys2/setup-msys2@v2
-        with:
-          msystem: MINGW${{ matrix.mingw-n-bits }}
-          update: true
-      - name: Setup MSYS2
-        shell: msys2 {0}
-        run: |
-          ci/scripts/msys2_setup.sh cgo
-      - name: Update CGO Env vars
-        shell: msys2 {0}
-        run: |
-          echo "CGO_CPPFLAGS=-I$(cygpath --windows ${MINGW_PREFIX}/include)" >> $GITHUB_ENV
-          echo "CGO_LDFLAGS=-g -O2 -L$(cygpath --windows ${MINGW_PREFIX}/lib) -L$(cygpath --windows ${MINGW_PREFIX}/bin)" >> $GITHUB_ENV
-          echo "MINGW_PREFIX=$(cygpath --windows ${MINGW_PREFIX})" >> $GITHUB_ENV
-      - name: Install go
-        uses: actions/setup-go@v3
-        with:
-          go-version: '1.18'
-          cache: true
-          cache-dependency-path: go/go.sum
-      - name: Install staticcheck
-        run: go install honnef.co/go/tools/cmd/staticcheck@latest
-      - name: Build
-        shell: bash
-        run: ci/scripts/go_build.sh $(pwd)
-      - name: Test
-        shell: bash
-        run: ci/scripts/go_test.sh $(pwd)
+# Licensed to the Apache Software Foundation (ASF) under one

```

Comment:
It seems that all lines are changed in this file. Is it intentional?

Reply:
CRLF  shenanigans on windows, sorry, will fix